### PR TITLE
Make candidate intake in society configurable

### PR DESCRIPTION
**Pull request by kianenigma**
Polkadot Companion: https://github.com/paritytech/polkadot/pull/2686

### CODE REVIEW

File: frame/society/src/lib.rs [hunk 1]
```diff
@@ -1615,11 +1621,12 @@ impl<T: Config<I>, I: Instance> Module<T, I> {
 	/// May be empty.
 	pub fn take_selected(
 		members_len: usize,
-		pot: BalanceOf<T, I>
+		pot: BalanceOf<T, I>,
 	) -> Vec<Bid<T::AccountId, BalanceOf<T, I>>> {
 		let max_members = MaxMembers::<I>::get() as usize;
 		// No more than 10 will be returned.
```
Comment (shawntabrizi) [inline frame/society/src/lib.rs:1627]: left over comment

**Review by shawntabrizi — APPROVED** [review]

**Review by coriolinus — APPROVED** [review]

File: frame/society/src/lib.rs (hunk 1 above)
Comment (kianenigma) [inline frame/society/src/lib.rs:1627]:
```suggestion
```

**Comment by kianenigma** [comment]
That was feaking fast @shawntabrizi @coriolinus

**Comment by coriolinus** [comment]
With 16 lines of total diff, none of which are complicated, it doesn't need to take long.

**Comment by kianenigma** [comment]
bot merge

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**